Resources/Prototypes/_Moffstation/Entities/Objects/Weapons/Melee/pirate_hook.yml

```
@@ -19,3 +19,25 @@
         Slash: 12
     soundHit:
       path: /Audio/Weapons/bladeslice.ogg
+
+- type: entity
+  parent: [ ClothingHandsBase ]
+  id: ClothingHandsShoddyPirateHook
+  name: shoddy pirate hook
+  description: Sharp and pointy, just a bit less than the real thing. Has a faded price tag sticker on the side depicting a spooky skeleton in a cloak.
+  suffix: Pirate
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Melee/pirate_hook.rsi
+    scale: 0.75, 0.75
+  - type: Clothing
+    sprite: _NF/Objects/Weapons/Melee/pirate_hook.rsi
+  - type: MeleeWeapon
+    autoAttack: true
+    animation: WeaponArcThrust
+    mustBeEquippedToUse: true
+    damage:
+      types:
+        Slash: 10
+    soundHit:
+      path: /Audio/Weapons/bladeslice.ogg
```

```suggestion
      path: /Audio/Weapons/bladeslice.ogg

```
The trailing newline will always finicky 😔